mysql-test/suite/innodb/r/innodb_mysql.result

```diff
@@ -386,10 +386,7 @@ name	dept
 rs5	cs10
 rs5	cs9
 DELETE FROM t1;
-# Masking (#) number in "rows" column of the following EXPLAIN output, as it may vary (bug#47746).
 EXPLAIN SELECT DISTINCT t1.name, t1.dept FROM t1 WHERE t1.name='rs5';
```

Comment:
Good catch - this is a --big-test, and the final patch version was not tested properly